src/app/components/buckets/buckets.scss

```diff
@@ -0,0 +1,184 @@
+@import 'variables';
+@import 'mixins';
+
+@mixin collapsed {
+    @media (max-width: 760px) {
+        @content;
+    }
+}
+
+.buckets-section {
+    padding: 3.8vw;
+    margin: 0 auto;
+    max-width: 120rem;
+
+    .bucket {
+        color: color(neutral-lightest);
+        display: flex;
+        margin: 0 auto 3.8vw;
+        max-width: 100%;
+        min-height: 30rem;
+        width: 90vw;
+
+        @include collapsed {
+            flex-direction: column;
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+
+        &.give {
+            background-color: color(green);
+
+            > .image {
+                background-image: url('/images/home/buckets/give.jpg');
+                background-position: center center;
+                background-repeat: no-repeat;
+                background-size: cover;
+            }
+
+            > .quote {
+                &::before {
+                    @include triangle(
+                        $direction: left,
+                        $position: top 27% left -2.4rem,
+                        $color: color(green),
+                        $size: 2.4rem
+                    );
+
+                    @include collapsed {
+                        @include triangle(
+                            $direction: top,
+                            $position: top -2.4rem left 17%,
```

Comment:
Per design, should be `right: 5rem` on mobile/collapsed
For refernene: homepage PDF Design file: https://drive.google.com/open?id=0BxscqVIq0m0tZjk0d1FZUGplaG8